examples/circular/circular_graph.py

```diff
@@ -26,9 +26,10 @@
 from dense.elements import Population
 
 
-pop = Population.from_swc(ds.NeuronsFromSimulation("circular_swc"))
+# pop = Population.from_swc(ds.NeuronsFromSimulation("circular_swc"))
+pop = Population.from_swc(ds.io.load_swc("circular_swc"))
```

Comment:
is this really how it works? shouldn't it be ``Population.from_swc("circular_swc")``?